lib/gui/models/store.js

```diff
@@ -394,10 +405,17 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
     }
 
     case ACTIONS.REMOVE_DRIVE: {
+      analytics.logEvent('Unselect drive', {
+        device: state.getIn([ 'selection', 'drive' ])
+      });
       return state.deleteIn([ 'selection', 'drive' ]);
     }
 
     case ACTIONS.REMOVE_IMAGE: {
+      const image = state.getIn([ 'selection', 'image' ]) && state.getIn([ 'selection', 'image' ]).toJS();
```

Comment:
I *think* this can be simplified to:

```js
const image = _.invoke(state.getIn([ 'selection', 'image' ]), 'toJS');
```

The `_.invoke` function will make sure `toJS` is not called if the image is undefined.